huaweicloud/services/acceptance/rds/resource_huaweicloud_rds_cross_region_backup_strategy_test.go

```diff
@@ -0,0 +1,140 @@
+	getBackupStrategyPath = strings.ReplaceAll(getBackupStrategyPath, "{instance_id}", state.Primary.ID)
+
+	getBackupStrategyOpt := golangsdk.RequestOpts{
+		KeepResponseBody: true,
+	}
+
+	getBackupStrategyResp, err := getBackupStrategyClient.Request("GET", getBackupStrategyPath, &getBackupStrategyOpt)
+	if err != nil {
+		return nil, fmt.Errorf("error retrieving RDS cross region backup strategy: %s", err)
+	}
+
+	getBackupStrategyRespBody, err := utils.FlattenResponse(getBackupStrategyResp)
+	if err != nil {
+		return nil, fmt.Errorf("error retrieving RDS cross region backup strategy: %s", err)
+	}
+
+	policyPara := utils.PathSearch("policy_para", getBackupStrategyRespBody, nil)
+	if policyPara == nil {
+		return nil, fmt.Errorf("error retrieving RDS cross region backup strategy: %s", err)
+	}
+
+	backupStrategies := policyPara.([]interface{})
+	if len(backupStrategies) == 0 || utils.PathSearch("keep_days", backupStrategies[0], 0).(float64) == 0 {
+		return nil, fmt.Errorf("error retrieving RDS cross region backup strategy: %s", err)
+	}
+
+	return getBackupStrategyRespBody, nil
+}
+
+func TestAccBackupStrategy_basic(t *testing.T) {
+	var obj interface{}
+
+	name := acceptance.RandomAccResourceName()
+	rName := "huaweicloud_rds_cross_region_backup_strategy.test"
+	dbPwd := fmt.Sprintf("%s%s%d", acctest.RandString(5),
+		acctest.RandStringFromCharSet(2, "!#%^*"), acctest.RandIntRange(10, 99))
+
+	rc := acceptance.InitResourceCheck(
+		rName,
+		&obj,
+		getBackupStrategyResourceFunc,
+	)
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:          func() { acceptance.TestAccPreCheck(t) },
+		ProviderFactories: acceptance.TestAccProviderFactories,
+		CheckDestroy:      rc.CheckResourceDestroy(),
+		Steps: []resource.TestStep{
+			{
+				Config: testBackupStrategy_basic(name, dbPwd),
+				Check: resource.ComposeTestCheckFunc(
+					rc.CheckResourceExists(),
+					resource.TestCheckResourceAttrPair(rName, "instance_id",
+						"huaweicloud_rds_instance.test", "id"),
+					resource.TestCheckResourceAttr(rName, "backup_type", "auto"),
+					resource.TestCheckResourceAttr(rName, "keep_days", "5"),
+					resource.TestCheckResourceAttr(rName, "destination_region", acceptance.HW_DEST_REGION),
+					resource.TestCheckResourceAttr(rName, "destination_project_id", acceptance.HW_DEST_PROJECT_ID),
+				),
+			},
+			{
+				Config: testBackupStrategy_basic_update(name, dbPwd),
+				Check: resource.ComposeTestCheckFunc(
+					rc.CheckResourceExists(),
+					resource.TestCheckResourceAttrPair(rName, "instance_id",
+						"huaweicloud_rds_instance.test", "id"),
+					resource.TestCheckResourceAttr(rName, "backup_type", "all"),
+					resource.TestCheckResourceAttr(rName, "keep_days", "8"),
+					resource.TestCheckResourceAttr(rName, "destination_region", acceptance.HW_DEST_REGION),
+					resource.TestCheckResourceAttr(rName, "destination_project_id", acceptance.HW_DEST_PROJECT_ID),
+				),
+			},
+			{
```

Comment:
suggest to append a basic config to the step 3 and checkout the modify logic for changing backup type and keep days.

Reply:
done